src/main/java/org/klodnicki/controller/BookController.java

```diff
@@ -69,32 +68,14 @@ public void add() throws IllegalArgumentException {
     }
 
     public void showBooks() {
-        menuController.displayOnMenu(LIST_OF_BOOKS_IN_DATABASE);
+
+        menuController.displayOnMenu(POSSIBLE_FIELDS_TO_SORT);
```

Comment:
What I do not like about this solution now is that you have this enum with all available sort options and you still decided to put it in another place. In this case you created `POSSIBLE_FIELDS_TO_SORT` which in case of changing sort options you will have to change too. And worse, you will have to remember about it!
That is a bad practice and you should always take that into consideration. Please, find a way to fix it, to have only possible sorting options in one place. For example enum, and display them (if you want to go that way) to user by using this enum. Then you won't need to remember about every String and where to change it and so on when any little change happens.

Reply:
thank you for that! thank you that you highlighted this to me so clearly! you are absolutely right so I made changes it should work much better now!